### PR TITLE
LabHub: Fix spelling of "eligility"

### PR DESCRIPTION
**Pull request by jayvdb**
Fixes https://github.com/coala/corobo/issues/329

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

**Comment by jayvdb** [comment]
unack https://github.com/coala/corobo/pull/472/commits/4c027c47b54412e01af87031fda345a51dbf1d09

File: plugins/labhub.py [hunk 1]
```diff
@@ -333,7 +333,8 @@ def eligible(user, iss):
                            'issue. :tada:')
                 else:
                     yield 'You are not eligible to be assigned to this issue.'
-                    yield '\n'.join(eligility_conditions)
+                    yield '\n'.join(
+                        bility_conditions)
```
Comment (andrewda) [inline plugins/labhub.py]: 🤔

Comment (andrewda) [reply plugins/labhub.py]: Wait how is this even pass tests?

Comment (Mixih) [reply plugins/labhub.py]: probably not being tested. That would be a separate issue though

Comment (andrewda) [reply plugins/labhub.py]: If it's not tested, wouldn't test coverage be < 100% and cause the code to fail CI?

Comment (meetmangukiya) [reply plugins/labhub.py]: Yeah, how does this work? This should raise `NameError` :confused:

Comment (jayvdb) [reply plugins/labhub.py]: autoflake isnt enabled?

Comment (jayvdb) [reply plugins/labhub.py]: fyi, this was a merge conflict.  His original patch was good, but needed rebasing.

**Comment by jayvdb** [comment]
Task abandoned, including after #474

**Pull request closed without merging**